src/options.json

```diff
@@ -6476,6 +6476,7 @@
         "title": "Max line length",
         "beautifiers": [
           "autopep8",
+          "black",
```

Comment:
Please complete the following:

- [x] Option added to Python language: https://github.com/Glavin001/atom-beautify/blob/master/src/languages/python.coffee#L23
- [ ] Option added to `Black` beautifier: https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/black.coffee#L29

See https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/js-beautify.coffee#L18-L25 for example:

```diff
-     Python: false
+    Python: 
+      wrap_line_length: true
```

- [ ] Re-generate the docs: https://github.com/Glavin001/atom-beautify/blob/master/docs/add-languages-and-beautifiers.md#options-documentation

```bash
npm run docs
```

Reply:
Should be done in 52301ec.